test/models/notification_preferences_test.rb

```diff
@@ -75,10 +75,28 @@ def test_hidden_preferences
   test 'enabled_notifications=' do
     preferences = NotificationPreferences.new
 
-    enabled = preferences.enabled_notifications = %w(application_created)
+    enabled = preferences.enabled_notifications = %w[application_created]
     enabled += hidden_notifications.map(&:to_s)
 
     assert_same_elements enabled, preferences.enabled_notifications
     assert_equal preferences.available_notifications, Set.new(preferences.preferences.keys)
   end
+
+  test 'new_preferences=' do
```

Comment:
Maybe it should test that all notification preferences not explicitly mentioned are not altered.

Reply:
I think I addressed it here: https://github.com/3scale/porta/pull/4080/commits/916e3d7151aa04a69a718b7669a4ab4349f03b70